openpype/tools/workfile_template_build/window.py

```diff
@@ -240,3 +253,68 @@ def showEvent(self, event):
             self._first_show = False
             self.setStyleSheet(style.load_stylesheet())
             self.resize(390, 450)
+
+    @QtCore.Slot(str, EnumAttrWidget)
+    def on_enum_change(
+        self,
+        value,
+        plugin,
+        origin_widget_label,
+        target_widget_label,
+        options=None
+    ):
+        self._clear_content_widget()
+        attr_defs = plugin.get_placeholder_options(options)
+
+        widget_to_modify = [
+            attr for attr in attr_defs if attr.label == target_widget_label
+        ]
+
+        new_content = self.get_representation_by_family(value)
+        widget_to_modify[0].items = new_content
+
+        widget = AttributeDefinitionsWidget(attr_defs, self._content_widget)
+
+        enum_widgets = widget.findChildren(EnumAttrWidget)
+        current_widget = [
+            w for w in enum_widgets if w.attr_def.label == origin_widget_label
+        ]
+        current_widget[0].set_value(value)
+
+        current_widget[0].value_changed.connect(
+            lambda value: self.on_enum_change(
+                value,
+                plugin,
+                origin_widget_label,
+                target_widget_label
+            )
+        )
+
+        self._content_layout.addWidget(widget, 0)
+        self._content_layout.addStretch(1)
+        self._attr_defs_widget = widget
+
+    def get_representation_by_family(self, family):
```

Comment:
```suggestion
    def get_representations_by_family(self, family):
```
Does it make more sense here to keep `representations` plural?

Reply:
Yes indeed it does